lib/get-config.js

```diff
@@ -283,6 +286,28 @@ function processIgnores(config) {
   config.ignore = config.ignore.map((pattern) => micromatch.matcher(pattern));
 }
 
+/**
+ * we were passed a rule, add the rule being passed in, to the config.
+ * ex:
+ * rule:severity
+ * no-implicit-this:["error", { "allow": ["some-helper"] }]
+ */
+function getRuleFromString(rule) {
+  const indexOfSeparator = rule.indexOf(':') + 1;
+
+  // we have to split based on the index of the first instance of the separator because the separator could exist in the second half of the rule
+  const name = rule.substring(0, indexOfSeparator - 1);
+  let ruleConfig = rule.substring(indexOfSeparator);
+
+  if (ruleConfig.startsWith('[')) {
+    ruleConfig = JSON.parse(ruleConfig);
```

Comment:
Thoughts on wrapping this with a try/catch to give a better error message when it is malformed?

Not sure this is "good" per-se, but something like this would maybe seem nicer:

```suggestion
    try {
      ruleConfig = JSON.parse(ruleConfig);
    } catch () {
      throw new Error(`Error parsing specified \`--rule\` config ${rule} as JSON.`);
    }
```

Reply:
Sounds good to me, I will add a test for this.